portfolio/src/main/java/com/google/sps/servlets/DailyActivitiesServlet.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import java.io.FileReader;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns daily activities data as a JSON object */
+@WebServlet("/daily-activities")
+public class DailyActivitiesServlet extends HttpServlet {
+  // Private variables used to construct json. Represents hours spend on each activity
+  private double eating, freeTime, reading, sleeping, working, exercising;
```

Comment:
What can we use to store multiples of the same item?  (In this case multiple activity hours).  This would make it easier to add/remove items later on.

Reply:
Ah of course! Thanks, haha